client/src/styles/components/dropdown-menu.css

```diff
@@ -20,15 +20,18 @@ Styleguide Components.Dropdown
   .dropdown-menu::before,
   .dropdown-menu::after {
     position: absolute;
-    top: -10px;
     left: 65px;
     border-right: 10px solid transparent;
     border-bottom: 10px solid var(--global-color-accent--normal);
     border-left: 10px solid transparent;
     margin-left: 20px;
     content: '';
   }
+  .dropdown-menu::before {
+    top: -10px;
+  }
   .dropdown-menu::after {
+    top: -9px;
     border-bottom-color: var(--global-color-primary--xx-light);
   }
```

Comment:
I tested, on Chromium-based browser, removing the erroneous `-10px`, and **not** adding the new values, fixes the bug (with less code). I will test other browsers.

```suggestion
  .dropdown-menu::after {
    border-bottom-color: var(--global-color-primary--xx-light);
  }
```

| remove new `:after` style | remove new `:before` style |
| - | - |
| <img width="960" alt="remove new after style" src="https://github.com/user-attachments/assets/e987ce88-d4eb-4a6b-990a-1f8a6ce1ff44"> |  <img width="960" alt="remove new before style" src="https://github.com/user-attachments/assets/cb3b1e6c-ffb2-4917-9b48-5756fa869e87">

Reply:
Tested on Safari via deploy to [dev.cep](https://dev.cep.tacc.utexas.edu/). Just removing the `-10px` seems adequate.

<details><summary>Video</summary>

https://github.com/user-attachments/assets/76ef9b49-5b8e-4da1-80f9-eae235fee0bc

</details>

Reply:
Opened jira [WP-791](https://tacc-main.atlassian.net/browse/WP-791) to track feedback